docs/develop/java/temporal-nexus.mdx

```diff
@@ -0,0 +1,654 @@
+
+Alternatively, you can create a Nexus Endpoint through the UI: [https://cloud.temporal.io/nexus](https://cloud.temporal.io/nexus).
+You can also create a Nexus endpoint through the UI in the development server.
+
+### Run Workers Connected to Temporal Cloud
+
+[View the source code](https://github.com/temporalio/samples-go/tree/main/nexus) in the context of the rest of the application code.
+
+Run the handler Worker:
+
+```go
+./gradlew -q execute -PmainClass=io.temporal.samples.nexus.handler.HandlerWorker \
+    --args="-target-host <your-target-namespace.account>.tmprl.cloud:7233 \
+	-namespace <your-target-namespace.account> \
+	-client-cert 'path/to/your/ca.pem' \
+	-client-key 'path/to/your/ca.key'"
+```
+
+Run the caller Worker:
+
+```go
+./gradlew -q execute -PmainClass=io.temporal.samples.nexus.caller.CallerWorker \
+    --args="-target-host <your-caller-namespace.account>.tmprl.cloud:7233 \
+	-namespace <your-caller-namespace.account> \
+	-client-cert 'path/to/your/ca.pem' \
+	-client-key 'path/to/your/ca.key'"
+```
+
+### Start a caller Workflow
+
+In order to start the caller Workflow, run the starter.
+
+```go
+./gradlew -q execute -PmainClass=io.temporal.samples.nexus.caller.CallerStarter \
+    --args="-target-host <your-caller-namespace.account>.tmprl.cloud:7233 \
+	-namespace <your-caller-namespace.account> \
+	-client-cert 'path/to/your/ca.pem' \
+	-client-key 'path/to/your/ca.key'"
+```
+
+This will result in:
+
+```
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Workflow result: Nexus Echo 👋 
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Started workflow workflowId: 9b3de8ba-28ae-42fb-8087-bdedf4cecd39 runId: 404a2529-764d-4d1d-9de5-8a9475e40fba 
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Workflow result: ¡Hola! Nexus 👋 
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Started workflow workflowId: 9cb29897-356a-4714-87b7-aa2f00784a46 runId: 7e71e62a-db50-49da-b081-24b61016a0fc
+```
+
+## Observability
+
+### Web UI
+
+A synchronous Nexus Operation will surface in the caller Workflow as follows, with just `NexusOperationScheduled` and `NexusOperationCompleted` events in the caller’s Workflow history:
+
+![Observability Sync](/img/nexus-go/go-sdk-observability-sync.png)
+
+An asynchronous Nexus Operation will surface in the caller Workflow as follows, with `NexusOperationScheduled`, `NexusOperationStarted`, and `NexusOperationCompleted`, in the caller’s Workflow history:
+
+![Observability Async](/img/nexus-go/go-sdk-observability-async.png)
+
+### Temporal CLI
+
+Use the `workflow describe` command to show pending Nexus Operations in the caller Workflow and any attached callbacks on the handler Workflow:
+
+```
+temporal workflow describe -w <ID>
+```
+
+Nexus events are included in the caller’s Workflow history:
+
+```
+temporal workflow show -w <ID>
+```
+
+For **asynchronous Nexus Operations** the following are reported in the caller’s history:
```

Comment:
Is there any interesting metadata of note to along with these events?

Reply:
I don't think so?